livy/src/main/java/org/apache/zeppelin/livy/BaseLivyInterprereter.java

```diff
@@ -206,7 +206,8 @@ private SessionInfo createSession(String user, String kind)
           conf.put(entry.getKey().toString().substring(5), entry.getValue().toString());
       }
 
-      CreateSessionRequest request = new CreateSessionRequest(kind, user, conf);
+      CreateSessionRequest request = new CreateSessionRequest(kind,
+          user.equals("anonymous") ? null : user, conf);
```

Comment:
Will this disable the proxyUser parameter to Livy? When anonymous mode is enabled in Zeppelin then the correct action seems to be to not pass proxyUser param since there is no proxy user.